src/sentry/static/sentry/app/actionCreators/projects.jsx

```diff
@@ -93,38 +93,96 @@ export function transferProject(api, orgId, project, email) {
 /**
  * Associate a team with a project
  */
-export function addTeamToProject(api, orgSlug, projectSlug, teamSlug) {
-  let endpoint = `/projects/${orgSlug}/${projectSlug}/teams/${teamSlug}/`;
+
+/**
+ *  Adds a team to a project
+ *
+ * @param {Client} api API Client
+ * @param {String} orgSlug Organization Slug
+ * @param {String} projectSlug Project Slug
+ * @param {String} teamSlug Team Slug
+ */
+export function addTeamToProject(api, orgSlug, projectSlug, team) {
+  let endpoint = `/projects/${orgSlug}/${projectSlug}/teams/${team.slug}/`;
+
+  addLoadingMessage();
```

Comment:
We never call this from action creators elsewhere, always directly from the component, intentional change?

Reply:
Yeah this is intentional, otherwise there's a slight delay when adding/removing a team to project